sdk/communication/Azure.Communication.CallingServer/tests/CallConnection/CallConnectionLiveTests.cs

```diff
@@ -62,6 +67,7 @@ public async Task RunCreatePlayCancelHangupScenarioTests()
             finally
             {
                 // Hang up the Call, there is one call leg in this test case, hangup the call will also delete the call as the result.
+                await WaitForOperationCompletion().ConfigureAwait(false);
```

Comment:
we dont need to wait for operation before hanging up. please remove this from other tests as well